straxen/plugins/multi_scatter.py

```diff
@@ -0,0 +1,325 @@
+import numba
+from straxen.plugins.position_reconstruction import DEFAULT_POSREC_ALGO
+from straxen.common import pax_file, get_resource, first_sr1_run, rotate_perp_wires
+from straxen.get_corrections import get_cmt_resource, is_cmt_option
+from straxen.itp_map import InterpolatingMap
+import strax
+import straxen
+import numpy as np
+export, __all__ = strax.exporter()
+
+@export
+class EventPeaks(strax.Plugin):
+    """
+    Add event number for peaks and drift times of all s2 depending on the largest s1.
+    Link - https://xe1t-wiki.lngs.infn.it/doku.php?id=weiss:analysis:ms_plugin
+    """
+    __version__ = '0.0.57'
+    depends_on = ('event_info', 'peak_basics','peak_positions')
```

Comment:
You don't need event_info, right? The lower level, the better especially since `event_info` is overwritten e.g. here:
https://github.com/XENONnT/cutax/blob/9e858f9504e00a7e92531149aef61b6dae0cf4f8/cutax/plugins/unblind_nr_tagged.py

Can you also make it work with 'event_basics' or 'events'?